template/public/index.html

```diff
@@ -15,11 +15,5 @@
       To begin the development, run `npm start`.
       To create a production bundle, use `npm run build`.
     -->
-    <script>
```

Comment:
I missed this - why are we removing the redirect?

Reply:
In order to show the explanation warning screen :)

Reply:
We've seen several people in the community being confused by the redirect. First, they don't understand that they shouldn't open `localhost` directly and if they do, they're completely lost with the target of the redirect. :)